raiden/tests/scenarios/join-network-scenario-J15.yaml

```diff
@@ -0,0 +1,39 @@
+settings:
+  gas_price: "fast"
+
+token:
+
+nodes:
+  commands:
+    stop: "pkill -f '127.0.0.1:50{:02d}'"
+    kill: "pkill -9 -f '127.0.0.1:50{:02d}'"
+    start: "raiden --config-file raiden_{0:02d}.toml --api-address 127.0.0.1:50{0:02d}"
+  range:
+    first: 1
+    last: 4
+    template: "localhost:50{:02d}"
+
+scenario:
+  serial:
+    tasks:
+      - parallel:
+          name: "Setting up a network"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 10, expected_http_status: 201}
+            - open_channel: {from: 0, to: 2, total_deposit: 10, expected_http_status: 201}
+            - open_channel: {from: 1, to: 2, total_deposit: 10, expected_http_status: 201}
+
+      - serial:
+          name: "Checking the network"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 10, balance: 10, state: "opened"}
+            - assert: {from: 0, to: 2, total_deposit: 10, balance: 10, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 10, balance: 10, state: "opened"}
+
+      - serial:
+          name: "Node Nr. 4 joins a network with 73 funds and joinable_funds set to 1"
+          tasks:
+            - join_network: {from: 3, funds: 73, joinable_funds_target: 1, expected_http_status: 204} # We expect 3 channels open with 0 tokens in it
```

Comment:
I probably don't get it, but why do we expect 0 deposits on the channels?

Reply:
joinable_funds_target is the part of funds that is being reserved for depositing into future channels, when new participants join the network. So `joinable_funds_target = 1` means 100% of the funds are being reserved and not deposited. Makes sense?

Reply:
Get it now, can you also add a TODO here

Reply:
Sure, you mean just 
TODO at the end?

Reply:
Yes, just add a comment below like: `TODO: add assertions here once partner selection is deterministic`